### DISCUSSION TITLE
Coe and Default Environment not visible

### DISCUSSION THREAD

**Discussion by leondarl** [microsoft/powerapps-tools#1041] [General]
Dear all,
 I am brand new with the Coe Kit but I love the idea of having it working in our tenant and track and maintain all Power Platform activities.
I have imported the Core Components on an "Trial" environment and I can only see the Environments I created. Somehow I cannot see the "Default" environment where all of our flows and apps are currently running.

Can you please help and guide me to properly troubleshoot the issue and resolve it.
Stay Safe !!
Thank you in advance
Leonidas

**Comment by rizqventure** [comment]
Yes please me too I need to be walked thru hand held
